doc/custom_conf.py

```diff
@@ -0,0 +1,162 @@
+        'ubuntu': 'https://discourse.ubuntu.com/t/'
+    },
+
+    # Change to the GitHub info for your project
+    'github_url': 'https://github.com/canonical/microcloud',
+
+    # Change to the branch for this version of the documentation
+    'github_version': 'main',
+
+    # Change to the folder that contains the documentation
+    # (usually "/" or "/docs/")
+    'github_folder': '/doc/',
+
+    # Change to an empty value if your GitHub repo doesn't have issues enabled.
+    # This will disable the feedback button and the issue link in the footer.
+    'github_issues': 'enabled',
+
+    # Controls the existence of Previous / Next buttons at the bottom of pages
+    # Valid options: none, prev, next, both
+    'sequential_nav': "both"
+}
+
+# If your project is on documentation.ubuntu.com, specify the project
+# slug (for example, "lxd") here.
+slug = ""
+
+############################################################
+### Redirects
+############################################################
+
+# Set up redirects (https://documatt.gitlab.io/sphinx-reredirects/usage.html)
+# For example: 'explanation/old-name.html': '../how-to/prettify.html',
+
+redirects = {}
+
+############################################################
+### Link checker exceptions
+############################################################
+
+# Links to ignore when checking links
+
+linkcheck_ignore = [
+    'http://127.0.0.1:8000'
+    ]
+
+# Pages on which to ignore anchors
+# (This list will be appended to linkcheck_anchors_ignore_for_url)
+
+custom_linkcheck_anchors_ignore_for_url = [
+    ]
+
+############################################################
+### Additions to default configuration
+############################################################
+
+## The following settings are appended to the default configuration.
+## Use them to extend the default functionality.
+
+# Add extensions
+custom_extensions = [
+    'sphinx.ext.intersphinx',
+]
+
+# Add files or directories that should be excluded from processing.
+custom_excludes = [
+]
+
+# Add CSS files (located in .sphinx/_static/)
+custom_html_css_files = []
+
+# Add JavaScript files (located in .sphinx/_static/)
+custom_html_js_files = []
+
+## The following settings override the default configuration.
+
+# Specify a reST string that is included at the end of each file.
+# If commented out, use the default (which pulls the reuse/links.txt
+# file into each reST file).
+# custom_rst_epilog = ''
+
+# By default, the documentation includes a feedback button at the top.
+# You can disable it by setting the following configuration to True.
+disable_feedback_button = False
+
+############################################################
+### Additional configuration
+############################################################
+
+## Add any configuration that is not covered by the common conf.py file.
+
+intersphinx_mapping = {
+    'lxd': ('https://documentation.ubuntu.com/lxd/en/latest/', None)
```

Comment:
is this right here?

Reply:
Yes, this is what we use to link to the LXD documentation. Intersphinx allows to use the Sphinx anchors directly - so instead of linking to `https://documentation.ubuntu.com/lxd/en/latest/`, we can link to `lxd:index`, for example. And we'd only need to update one location if we decided to move servers again. ;)

Reply:
excellent thanks